pkg/sqlcache/schema.go

```diff
@@ -0,0 +1,78 @@
+package sql
```

Comment:
I would prefer to see this in a sub-package called something like `schematracker` as in `pkg/sqlcache/schematracker`. It feels wrong to put it at the top-level `pkg/sqlcache` when we have no other code at that level, just `integration_test.go`